packages/frontend/src/main/components/stream/CollaboratorsDisplay.vue

```diff
@@ -20,7 +20,7 @@
         <span class="caption">+{{ collaborators.length - 4 }}</span>
       </v-btn>
       <v-btn
-        v-if="collaborators.length <= 5"
+        v-if="stream.role === Roles.Stream.Owner && collaborators.length <= 5"
```

Comment:
Hiding "Manage collaborators" button from streams toolbar if you're not the owner